### PR TITLE
writeMatplotlib

### LINKED ISSUE #50: writeMatPlotLib()
Concerning  https://github.com/craffael/lehrfempp/pull/49#issue-217375329, item 3:

Great idea to write a function 
~~~
void writeMatPlotLib(const lf::mesh::Mesh &mesh, std::string filename);
~~~
along the lines of `lf::mesh::utils::writeMatlab()`. It should produce an output that can be read by a Python script, which takes care of the visualization.

### PR DESCRIPTION
**Pull request by anianruoss**
To resolve #50 I implemented a MVP for plotting meshes with Python. While doing so, I realized that matplotlib is not very suited for this task. This is what a matplotlib mesh currently looks like:
screenshot from 2018-10-07 19-23-54

Furthermore, I found this neat [TikZ editor](http://www.tikzedt.org) which makes visualizing meshes generated by writeTikZ() very easy. 

For these two reasons I would suggest that we don't provide any matplotlib plotting at all and focus on TikZ and ParaView. Please let me know what you think.

### CODE REVIEW

File: lib/lf/mesh/utils/write_matplotlib.cc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                            // to access points of cell use SubEntities(1)
+                            for (const auto &sub: obj.SubEntities(codim + 1)) {
+                                file << mesh.Index(sub) << ',';
+                            }
+                            file << std::endl;
+
+                            break;
+                        }
+                        default: {
+                            std::cerr << "Error for object " << obj_idx
```
Comment (craffael) [inline lib/lf/mesh/utils/write_matplotlib.cc]: Since this case really should not happen, I would rather use a LF_VERIFY_MSG(false, ...).

**Comment by hiptmair** [comment]
- Judging from the screenshot, the output from matplotlib is really satisfactory. It is always good to have another option. 
- The TikzEdt editor has been developed by Prof. T. Willwacher, D-MATH, ETHZ, but since four years development has stalled and my colleague does not know whether and when he will resume.

File: lib/lf/mesh/utils/write_matplotlib.cc (hunk 1 above)
Comment (anianruoss) [reply lib/lf/mesh/utils/write_matplotlib.cc]: Are triangles and quadrilaterals the only cell shapes that we will support?

**Comment by anianruoss** [comment]
@craffael ptal

**Comment by craffael** [comment]
Looks good to me, thanks!

**Pull request merged**